### DISCUSSION TITLE
HostFS + File Sharing?

### DISCUSSION THREAD

**Discussion by IngwiePhoenix** [DustinBrett/daedalOS#420] [Feature Requests]
Hello there!

I already have a 10TB NAS set up with all my files structured and laid out like a gigantic home folder - because...its a NAS, my stuff goes there. Thing is, I have been wanting a way to easily share them but also use them when I am away. One way would be to use a web desktop that just mounts this NAS as the "user's" home folder. That shouldn't be too difficult.

The other thing is, that I would love to be able to share some files from within the desktop as a link to my collegues on occasion. Is that possible?

Thanks and kind regards!

PS. Nostr is moving to NIP-44 and friends for better encrypted DM support. ;)

**Comment by DustinBrett** [comment]
Hi @IngwiePhoenix,

It would indeed be nice to be able to mount a local file system. It's not impossible to do this, but the folder would need to be inside `public` or possibly just linked. Then the JSON builder could map it.

And indeed I have plans to upgrade Nostr also, I just haven't taken the time to make the adjustments. I'd posted about it a while ago. https://github.com/nbd-wtf/nostr-tools/issues/311
